### DISCUSSION TITLE
What is the CPE for Mirth Connect

### DISCUSSION THREAD

**Discussion by jsdratm** [nextgenhealthcare/connect#5847] [General]
Hello, I am including Mirth Connect in the SBOM for a product as a dependency and I was wondering what the CPE (common platform enumeration) is for this product.  I couldn't find any published vulnerabilities against Mirth Connect, so I wanted to see if there is a defined CPE that the company will use to publish any future CVEs against so we can list it in our software bill of materials appropriately.

https://nvd.nist.gov/products/cpe#:~:text=CPE%20is%20a%20structured%20naming,and%20tests%20to%20a%20name.

**Comment by jonbartels** [comment]
Related discussion in Slack - https://mirthconnect.slack.com/archives/CN8AS6QTG/p1689357856356369

The MCAL setup launcher cites additional software distributed under the Apache License. This list is out of date. 

I believe this is related to the issue raised by jsdratm because it suggests that a CPE or SBOM existed at one time but is simply out dated and needs revised.

**Comment by ChristopherSchultz** [comment]
> Hello, I am including Mirth Connect in the SBOM for a product as a dependency

How wonderfully progressive. I'd love to hear how this goes.

**Comment by jonbartels** [comment]
https://github.com/nextgenhealthcare/connect/issues/5852
